modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -2695,6 +2695,91 @@
 		<tag-class>com.liferay.frontend.taglib.clay.servlet.taglib.TabsPanelTag</tag-class>
 		<body-content>JSP</body-content>
 	</tag>
+	<tag>
+		<description></description>
+		<name>toggle</name>
+		<tag-class>com.liferay.frontend.taglib.clay.servlet.taglib.ToggleTag</tag-class>
+		<body-content>empty</body-content>
+		<attribute>
+			<description></description>
+			<name>cssClass</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<description></description>
+			<name>disabled</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<description></description>
+			<name>helpText</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<description></description>
+			<name>id</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<description></description>
+			<name>label</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<description></description>
+			<name>labelOff</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
+		<attribute>
+			<description></description>
+			<name>labelOn</name>
+			<required>false</required>
+			<rtexprvalue>true</rtexprvalue>
+		</attribute>
```

Comment:
[naming nit] These should be `offLabel` and `onLabel`, following general convention that last word(s) determine what the variable _is_, and prefix(es) add specificity and description.

The same with `offSymbol` and `onSymbol`.

Reply:
That makes sense, I originally name it this way so that all the label and symbol attributes would be grouped together. I'll follow the typical convention though and make this change.